h5py/h5p.pyx

```diff
@@ -1197,16 +1193,20 @@ cdef class PropFAID(PropInstanceID):
 
     IF MPI:
         @with_phil
-        def set_fapl_mpio(self, Comm comm not None, Info info not None):
+        def set_fapl_mpio(self, comm, info):
             """ (Comm comm, Info info)
 
             Set MPI-I/O Parallel HDF5 driver.
 
             Comm: An mpi4py.MPI.Comm instance
             Info: An mpi4py.MPI.Info instance
             """
-            H5Pset_fapl_mpio(self.id, comm.ob_mpi, info.ob_mpi)
-
+            from mpi4py import MPI
```

Comment:
Look at the failures on Travis. `MPI` is also a `DEF` (Cython preprocessor) variable.

Maybe you should do `import mpi4py.MPI`, or `from mpi4py.MPI import Comm, Info, _handleof`